cves/CVE-2011-3923.yml

```diff
@@ -213,4 +212,14 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    I feel as if the mistakes that were made that led to this vulnerability
+    were based in coding. The developers simply forgot to handle an edge case,
+    the edge case being the presence of parentheses to construct and make OGLN 
+    calls. The chosen CWE for this vulnerability (CWE-94) marks input validation
+    and testing as a couple of the possible mitigations, and in the case of this
+    CVE, the struts development team were already doing both, just not to the 
+    degree that was required to handle this vulnerability. Tests existed for this
+    area of the code and some other characters were also blacklisted from requests,
+    just not the characters that were needed to fully elminiate the possbility of
+    malicious OGLN code execution.
```

Comment:
There are a few run on sentences here, but they read well enough. I would consider breaking "The developers simply forgot to handle an edge case, the edge case being the presence of parentheses to construct" into two sentences or changing the second clause to "which was the presence of parentheses"

Reply:
This is in reference to the entire "Mistakes" section, I wasn't sure how to include all of that in the preview.

Reply:
I'll give the mistakes section another pass for readability/grammar and update where needed. Thanks!